MercadoPagoSDK.Test/Resources/PreapprovalTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿using MercadoPago;
+using MercadoPago.Resources;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Net;
+using Newtonsoft.Json.Linq;
+
+namespace MercadoPagoSDK.Test.Resources
+{
+    [TestFixture()]
+    public class PreapprovalTest
+    {
+        Preapproval LastPreapproval;
+
+        [SetUp]
+        public void Init()
+        {
+            // Avoid SSL Cert error
+            ServicePointManager.ServerCertificateValidationCallback = delegate { return true; };
+            // Environment.GetEnvironmentVariable("ACCESS_TOKEN");  
+            SDK.CleanConfiguration();
+            SDK.SetBaseUrl("https://api.mercadopago.com");
+            SDK.ClientId = Environment.GetEnvironmentVariable("CLIENT_ID");
+            SDK.ClientSecret = Environment.GetEnvironmentVariable("CLIENT_SECRET");
+        }
+
+        [Test]
+        public void Preapproval_CreateShouldBeOk()
+        {
+
+
+            Preapproval preapproval = new Preapproval()
+            {
+                Payer_email = "test@mail.com",
+                Back_url = "https://localhost",
```

Comment:
BackUrl should have the format ` /http[s]*\/\/[a-z0-9\-\_\.]\.[a-z0-9]/` : https://local.mydomain.com